packages/core/src/monaco-util.ts

```diff
@@ -1,7 +1,6 @@
 import 'monaco-editor/esm/vs/editor/editor.all.js';
 
 // support all editor features
-import 'monaco-editor/esm/vs/editor/standalone/browser/accessibilityHelp/accessibilityHelp.js';
```

Comment:
From what I can tell, all the imports were unified in a single import in https://github.com/TypeFox/monaco-languageclient/commit/2760e5d6324158421b7ac1bc37eaccbcd4f7a25b 
```
import 'monaco-editor/esm/vs/editor/edcore.main.js';
```

and then further adapted so I believe that now most of those services and features are used by default when you call `initServices` and we no longer have to explicitly add them.

Reply:
Ah ok I need to check that, then I can maybe remove more of those here...